rero_ils/static/scss/rero_ils/frontpage.scss

```diff
@@ -18,16 +18,22 @@
  */
 
  .rero-ils-frontpage {
-  height: 45vh;
   background-image: url(#{$static-images}/homepage_image_bleu.jpg);
   background-size: cover;
   background-position: center;
   form {
     margin: 0 auto;
     width: 80%;
   }
-  @include media-breakpoint-down(xs) {
-    height: 60vh;
+
+  @media (max-height: 500px){
```

Comment:
can you use the bootstrap mechanism? https://getbootstrap.com/docs/4.3/layout/overview/#responsive-breakpoints

Reply:
The bootstrap responsive breakpoints consider the width of the screen. But we want to adapt  to the height.